### DISCUSSION TITLE
How do we switch LPARs?

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#793] [IBM i] [answered]
I work with a production and development LPAR. I worked on production today and now need to switch back to development. I cannot find an easy way to switch over. I have to shutdown and restart.

Is there a simple way to do this?

TIA, Darryl Freinkel.

**Comment by worksofliam** [comment, marked as answer]
@dhfreinkel Yes!

Open the command palette (F1 or Control/Command + Shift + P) and search for 'disconnect'. Taking the disconnect option will disconnect you.

It also also possible to have two instances of Visual Studio Code open (Control/Command + Shift + N), which would allow you to have two different connections at once.
